modules/api/src/main/java/com/intuit/wasabi/api/ApiModule.java

```diff
@@ -101,6 +87,38 @@ protected void configure() {
 //            e.printStackTrace();
 //        }
 
-        LOGGER.debug("installed module: {}", ApiModule.class.getSimpleName());
+    }
+    
+    protected void installUserModule() {
```

Comment:
I am assuming you are doing this because you want to overwrite the behavior when intuit specific class going to install different modules. Isn't that already controlled by the properties file?